bots/example_bots/helpers/grid.py

```diff
@@ -88,19 +88,36 @@ def __init__(
     # inventory functions
     ######################################################################
 
+    def update_fair_price(self, fair_price: Decimal):
```

Comment:
it may be helpful to have the ability to return back whatever the difference between the existing grid and the new grid is. however, without tracking what existing offers there are on the market this may not be all that helpful within the `Grid` class itself.